src/main/kotlin/org/codefreak/codefreak/service/TaskTarService.kt

```diff
@@ -233,7 +234,8 @@ class TaskTarService : BaseService() {
         EvaluationDefinition(
             it.runnerName,
             it.options,
-            it.title
+            it.title,
+            it.active
```

Comment:
Only explicit `false` should be exported IMO.
Line 110 defaults to `true` anyways.